projects/plugins/jetpack/modules/widgets/top-posts.php

```diff
@@ -419,9 +419,13 @@ public function widget( $args, $instance ) {
 							'avatar_size'         => (int) $get_image_options['avatar_size'],
 						)
 					);
-					$post['image'] = isset( $image['src'] ) ? $image['src'] : null;
-					if ( isset( $image['from'] ) && 'blavatar' !== $image['from'] && 'gravatar' !== $image['from'] ) {
-						$post['image'] = jetpack_photon_url( $post['image'], array( 'resize' => "$width,$height" ) );
+
+					if ( ! empty( $image ) ) {
```

Comment:
`! empty()` does two things, it's basically `isset( $image ) && $image`. But we already know `$image` is set, so we don't need the first bit (and, in fact, it could accidentally hide it if you typoed to `$imgae` or something). Better to just do like
```suggestion
					if ( $image ) {
```

(sorry, this is one of my pet peeves)

Reply:
Thanks for the suggestion @anomiex . I have updated all the $image references.